staging/src/kubevirt.io/api/pool/v1alpha1/types.go

```diff
@@ -170,7 +175,49 @@ type VirtualMachinePoolSelectionPolicy struct {
 	// +optional
 	// +kubebuilder:validation:Enum=Random;DescendingOrder
 	BasePolicy *VirtualMachinePoolBasePolicy `json:"basePolicy,omitempty"`
+
+	// OrderedPolicies is a Ordered list of selection policies.
+	// +optional
+	OrderedPolicies *VirtualMachinePoolOrderedPolicy `json:"orderedPolicies,omitempty"`
+}
+
+// VirtualMachinePoolOrderedPolicy specifies filtering criteria for VM selection.
+// Both LabelSelector and NodeSelectorRequirementMatcher can be specified together.
```

Comment:
In the situation that they're both provided, do they have an `AND` or `OR` semantics? i.e. does a VM need to have both, or just one of them?